src/status_im/ui/screens/profile/visibility_status/views.cljs

```diff
@@ -25,17 +25,11 @@
   (re-frame/dispatch
    [:visibility-status-updates/delayed-visibility-status-update status-type]))
 
-;; In new ui, we are allowing switcher to overlap status-bar (draw over status bar)
-;; that's why the measure will return height including, the height of the status bar in android
-;; for calculating the correct position of the button on the profile screen, we have to decrease this height
 (defn calculate-button-height-and-dispatch-popover []
   (.measure
    @button-ref
    (fn  [_ _ _ _ _ py]
```

Comment:
can we consider using named parameters instead? also what is `py`?

Reply:
hi @J-Son89, Thank you for reviewing PR.
py is pageY ([ref](https://github.com/facebook/react-native/blob/65975dd28de0a7b8b8c4eef6479bf7eee5fcfb93/Libraries/Renderer/shims/ReactNativeTypes.js#L18)) (Y offset to page)

But for some reason people usually prefer py. [Example 1](https://stackoverflow.com/a/32751446) [Example 2](https://stackoverflow.com/q/29828971)

changed to page-y ( but not sure if it still  understandable just by name :) )

Reply:
@qoqobolo, Please ignore latest commit, it will not affect testing (its only renames one variable)

Reply:
> but not sure if it still understandable just by name :)

still it is better than py :)